etc/kayobe/ansible/vault-generate-backend-tls.yml

```diff
@@ -1,10 +1,42 @@
 ---
+# Required for uri module to work with self-signed certificates and for systems to trust
+# the self-signed CA
+- name: Install CA on network hosts
```

Comment:
Network hosts? Target is controllers

Reply:
> Network hosts? Target is controllers

My mistake, I forgot to edit when I removed `network` hosts from my changes.

Reply:
> Maybe we need to expand the target hosts in vault-deploy-overcloud.yml?

I have a preference for having explicit dependencies, otherwise we can run into issues when nodes are reprovisioned for example.